src/Umbraco.Cms.Integrations.Testsite.V14/Umbraco.Cms.Integrations.Testsite.V14.csproj

```diff
@@ -4,6 +4,14 @@
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>enable</Nullable>
   </PropertyGroup>
+  <ItemGroup>
```

Comment:
I include these 2 files `semrushColumns.json` and `semrushDataSources.json` in the test site, because I'm not sure where to put them properly. wwwroot or in `Umbraco.Cms.Integrations.SEO.Semrush` project

Reply:
@acoumb I don't remember what the reason for these files were... but if it was to allow users to update them, that won't be possible now they are coming from a Razor Class Library.  So maybe their contents should be removed and added as constants in the code (if there's no need for customers to update), or as configuration (if there is).

Reply:
I see they are quite big though... so not ideal in configuration or constants.  Perhaps embedded resources is the right way to go, used as a fallback having first checked for a file on disk if the customer has provided it?

Reply:
The files are used to display the tooltips in the results table. So you can either use them from the JSON files, load them server-side or cache them.

Reply:
I picked the data from their API [docs](https://developer.semrush.com/api/v3/analytics/basic-docs/).